src/ZoneServer/ZoneServer.cs

```diff
@@ -124,8 +128,21 @@ public override void Run(string[] args)
 			this.StartCommunicator();
 			this.StartAcceptor();
 
-			ConsoleUtil.RunningTitle();
-			new ConsoleCommands().Wait();
+
+			// If running on Windows system, enable console inputs
+			// otherwise, start an event loop to keep server running until stopped
+			if (Environment.OSVersion.Platform == PlatformID.Win32NT)
```

Comment:
This seems like a suboptimal thing to do, because if I'm running the server in a console under Linux, I might want console commands. In the same vein, I might run the server without a console on Windows. It seems to me like a check for an interactive environment would be more appropriate.

We actually have a property for that on `ConsoleUtil`, "`IsUserInteractive`", though I'll admit this property hasn't been used or confirmed to still be working in a while.